OpenTrons/opentrons-api
# refactor(odd): update the welcome screen

# Overview
update the welcome screen
close RCORE-748

![Screenshot 2023-04-28 at 9 31 11 AM](https://user-images.githubusercontent.com/474225/235165695-a9f76fec-99ce-4e54-a2ba-cf0ac2455855.png)


[design]
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/Release%3A-Opentrons-Flex-Touchscreen-Designs?node-id=7270-116000&t=GouSuVbgIM4rBv2g-0


# Test Plan
This should be testable by a dev kit/flex or on-device-mode
1. Tap ODD menu
2. Tap Welcome button


# Changelog
- change the route for Welcome component `/get-started` -> `/welcom`
- update test for Welcome component
- add test for Welcome component to OnDeviceDisplayApp


# Review requests



# Risk assessment
low
